### PR TITLE
Add model setting completion params for Image2Text prompt schema

### PR DESCRIPTION
**Pull request by rossdanlm**
Add model setting completion params for Image2Text prompt schema

TSIA, pretty simple actually

## Test Plan
Follow the README from AIConfig Editor https://github.com/lastmile-ai/aiconfig/tree/main/python/src/aiconfig/editor#dev, then run these commands:

Make sure also that you don't have the `python-aiconfig-test` installed: `pip3 uninstall python-aiconfig-test`

```bash
aiconfig_path=/Users/rossdancraig/Projects/aiconfig/cookbooks/Gradio/huggingface.aiconfig.json
parsers_path=/Users/rossdancraig/Projects/aiconfig/cookbooks/Gradio/hf_model_parsers.py
alias aiconfig="python3 -m 'aiconfig.scripts.aiconfig_cli'"
aiconfig edit --aiconfig-path=$aiconfig_path --server-port=8080 --server-mode=debug_servers --parsers-module-path=$parsers_path
```

Screenshot 2024-01-11 at 01 21 21

---
Stack created with [Sapling](https://sapling-scm.com). Best reviewed with [ReviewStack](https://reviewstack.dev/lastmile-ai/aiconfig/pull/875).
* #877
* __->__ #875

### CODE REVIEW

File: python/src/aiconfig/editor/client/src/shared/prompt_schemas/HuggingFaceAutomaticSpeechRecognitionPromptSchema.ts [hunk 1]
```diff
@@ -15,7 +15,14 @@ export const HuggingFaceAutomaticSpeechRecognitionPromptSchema: PromptSchema = {
...
           type: "attachment",
           required: ["data"],
-          mime_types: ["audio/mpeg", "audio/wav", "audio/webm", "audio/flac", "audio/ogg", "audio/ogg"],
+          mime_types: [
+            "audio/mpeg",
+            "audio/wav",
+            "audio/webm",
+            "audio/flac",
+            "audio/ogg",
+            "audio/ogg",
```
Comment (saqadri) [inline python/src/aiconfig/editor/client/src/shared/prompt_schemas/HuggingFaceAutomaticSpeechRecognitionPromptSchema.ts:24]: This looks duplicated. Error?

File: python/src/aiconfig/editor/client/src/shared/prompt_schemas/HuggingFaceAutomaticSpeechRecognitionPromptSchema.ts [hunk 2]
```diff
@@ -65,15 +72,12 @@ export const HuggingFaceAutomaticSpeechRecognitionPromptSchema: PromptSchema = {
...
         type: "string",
         enum: ["word", "char", "True", ""],
-        description: `Only available for pure CTC models (Wav2Vec2, HuBERT, etc) and the Whisper model. Not available for other sequence-to-sequence models.`
+        description: `Only available for pure CTC models (Wav2Vec2, HuBERT, etc) and the Whisper model. Not available for other sequence-to-sequence models.`,
       },
       max_new_tokens: {
-        type: "number",
-        description: `The maximum numbers of tokens to generate, ignoring the number of tokens in the prompt`
-      }
+        type: "integer",
```
Comment (saqadri) [inline python/src/aiconfig/editor/client/src/shared/prompt_schemas/HuggingFaceAutomaticSpeechRecognitionPromptSchema.ts:78]: Why this change?

File: python/src/aiconfig/editor/client/src/shared/prompt_schemas/HuggingFaceImage2TextTransformerPromptSchema.ts [hunk 3]
```diff
@@ -28,6 +32,16 @@ export const HuggingFaceImage2TextTransformerPromptSchema: PromptSchema = {
...
+      max_new_tokens: {
+        type: "integer",
+        description: `The amount of maximum tokens to generate. 
+        By default it will use \`generate\` default.`,
+      },
+      timeout: {
+        type: "number",
+        description: `The maximum time in seconds to wait for fetching images 
+        from the web. If None, no timeout is set and the call may block forever.`,
+      },
```
Comment (saqadri) [inline python/src/aiconfig/editor/client/src/shared/prompt_schemas/HuggingFaceImage2TextTransformerPromptSchema.ts:44]: Oh wow that's a short list of possible params

**Review by saqadri — APPROVED** [review]

File: python/src/aiconfig/editor/client/src/shared/prompt_schemas/HuggingFaceAutomaticSpeechRecognitionPromptSchema.ts (hunk 1 above)
Comment (rossdanlm) [reply python/src/aiconfig/editor/client/src/shared/prompt_schemas/HuggingFaceAutomaticSpeechRecognitionPromptSchema.ts:24]: It's just auto-formatting when I hit the CMD + S

File: python/src/aiconfig/editor/client/src/shared/prompt_schemas/HuggingFaceAutomaticSpeechRecognitionPromptSchema.ts (hunk 2 above)
Comment (rossdanlm) [reply python/src/aiconfig/editor/client/src/shared/prompt_schemas/HuggingFaceAutomaticSpeechRecognitionPromptSchema.ts:78]: Because number is a float while integer is an integer value. I think this was incorrectly transcribed the first time

File: python/src/aiconfig/editor/client/src/shared/prompt_schemas/HuggingFaceImage2TextTransformerPromptSchema.ts (hunk 3 above)
Comment (rossdanlm) [reply python/src/aiconfig/editor/client/src/shared/prompt_schemas/HuggingFaceImage2TextTransformerPromptSchema.ts:44]: yea it's actually pretty wild. It's why I left a comment about it in in summary for https://github.com/lastmile-ai/aiconfig/pull/855
